proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -155,6 +155,7 @@ message MsgUndelegate {
 message MsgUndelegateResponse {
   google.protobuf.Timestamp completion_time = 1
       [(gogoproto.nullable) = false, (amino.dont_omitempty) = true, (gogoproto.stdtime) = true];
+  cosmos.base.v1beta1.Coin amount            = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
```

Comment:
```suggestion
  // amount returns the amount undelegated coins
  //
  // Since: cosmos-sdk 0.48
  cosmos.base.v1beta1.Coin amount            = 2 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];
```

This is required. Could you re-generate the proto after adding the since comment